src/placement-center/src/raftv1/rocksdb.rs

```diff
@@ -355,7 +355,7 @@ impl RaftMachineStorage {
         }
     }
 
-    #[warn(dead_code)]
+    #[allow(dead_code)]
```

Comment:
Similarly, is this supposed to be #[allow(dead_code)] instead of #[warn(dead_code)]?